src/messages/MClientRequest.h

```diff
@@ -191,6 +193,7 @@ class MClientRequest final : public MMDSOp {
   }
 
   utime_t get_stamp() const { return stamp; }
+  ceph::coarse_mono_time get_coarse_stamp() const { return coarse_stamp; }
```

Comment:
This makes no sense, the `coarse_stamp` will only be used in the client side, no need to pass this to MDS.